esp/scm/ws_workunits.ecm

```diff
@@ -1123,6 +1144,19 @@ ESPresponse [exceptions_inline] WUQueryGetGraphResponse
     ESParray<ESPstruct ECLGraphEx> Graphs;
 };
 
+ESPrequest WUQueryGetSummaryStatsRequest
+{
+    string Target;
+    string QueryId;
+    string FromTime; //YYYY-MM-DDTHH:MM:SS
+    string ToTime;
+};
+
+ESPresponse [exceptions_inline] WUQueryGetSummaryStatsResponse
+{
+    ESParray<ESPstruct QuerySummaryStats> QuerySummaryStatsList;
```

Comment:
Don't really need "QuerySummary" in the field name, since it's implied by being in the QuerySummaryStatsResponse.